devpack/src/main/java/io/neow3j/devpack/Storage.java

```diff
@@ -46,6 +46,56 @@
     @Instruction(interopService = SYSTEM_STORAGE_GET)
     public static native ByteString get(StorageContext context, byte[] key);
 
+    /**
+     * Returns the value corresponding to the given key and converts it to a byte array.
+     * <p>
+     * This incurs the GAS cost of converting the {@code ByteString} value to a byte array.
+     *
+     * @param context The storage context to search in.
+     * @param key     The key to search for.
+     * @return the value corresponding to the given key converted to a byte array.
+     */
+    @Instruction(opcode = OpCode.SWAP)
+    @Instruction(interopService = SYSTEM_STORAGE_GET)
+    @Instruction(opcode = OpCode.CONVERT, operand = StackItemType.BUFFER_CODE)
+    public static native byte[] getByteArray(StorageContext context, byte[] key);
+
+    /**
+     * Returns the value corresponding to the given key as a string.
+     * <p>
+     * This does not incur any extra GAS costs.
+     *
+     * @param context The storage context to search in.
+     * @param key     The key to search for.
+     * @return the value corresponding to the given key as a string.
+     */
+    @Instruction(opcode = OpCode.SWAP)
+    @Instruction(interopService = SYSTEM_STORAGE_GET)
+    @Instruction()
+    public static native String getString(StorageContext context, byte[] key);
+
+    /**
+     * Returns the value corresponding to the given key and converts it to an integer. The bytes
+     * are read in little-endian format. E.g., the byte string {@code 0102} (in hexadecimal
+     * representation) is converted to 513.
+     * <p>
+     * This incurs the GAS cost of converting the {@code ByteString} value to an integer.
```

Comment:
Please mention that it returns an int value of 0 if the key is not found.

Or maybe we should not return 0 but actually null because 0 is a valid integer value and could be misleading. I mean, just because 0 is returned doesn't necessarily mean that no value was found. I.e., 0 is ambiguous. What do you think?

Reply:
I changed the instructions to return `null` instead of `0` if the value is `null`. I adapted the according methods in the classes `Helper` (`toInt()`) and `ByteString` (`toInteger()`) as well.